tools/assist.py

```diff
@@ -1,19 +1,17 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
-import argparse
+import argparse, os, sys
```

Comment:
It is better style to import different modules on different lines

Why? Because it was said so
https://peps.python.org/pep-0008/#imports
(there are various reasons but style consistency and sticking to a standard beats everything)